src/v/config/configuration.cc

```diff
@@ -3659,6 +3659,15 @@ configuration::configuration()
        tls_version::v1_1,
        tls_version::v1_2,
        tls_version::v1_3})
+  , tls_enable_renegotiation(
+      *this,
+      "tls_enable_renegotiation",
+      "TLS client renegotiation was removed in TLSv1.3 due to vulnerabilities "
+      "found in the protocol.  By default, this is disabled for TLSv1.2 and "
+      "below.  It can be re-enabled by setting this property to true and "
+      "restarting Redpanda.  This is considered unsafe.",
```

Comment:
```suggestion
      "TLS client renegotiation, removed in TLSv1.3 due to vulnerabilities, "
      "is disabled for TLSv1.2 and earlier. "
      "It can be re-enabled by setting this property to true and "
      "restarting Redpanda.  This is considered unsafe.",
```